lte/gateway/python/magma/pipelined/app/restart_mixin.py

```diff
@@ -28,6 +35,17 @@ class RestartMixin(metaclass=ABCMeta):
 
     Mixin class for controller restart handling
     """
+    logger: Logger
+    tbl_num: int
+    cleanup_state: Callable
+    delete_all_flows: Callable
+    finish_init: Callable
+    _msg_hub: MessageHub
+    _datapath: Datapath
+    _startup_flow_controller: Optional[StartupFlows]
+    _startup_flows_fut: Future[StartupFlows]
+    _clean_restart: Callable
```

Comment:
Is `_clean_restart` definitely callable? It seems to me that it usually gets set to the `clean_restart` part of the config, which is a boolean.

Reply:
Good catch, that's an error.

(And it's bad that mypy didn't notice...)